src/testFixtures/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/database/jpa/entity/TimelineEntityAssert.kt

```diff
@@ -0,0 +1,89 @@
+package uk.gov.justice.digital.hmpps.educationandworkplanapi.app.database.jpa.entity
+
+import org.assertj.core.api.AbstractObjectAssert
+import java.time.Instant
+import java.util.UUID
+import java.util.function.Consumer
+
+fun assertThat(actual: TimelineEntity?) = TimelineEntityAssert(actual)
+
+/**
+ * AssertJ custom assertion for [TimelineEntity]
+ */
+class TimelineEntityAssert(actual: TimelineEntity?) :
```

Comment:
Not actually used in this PR. I added this as I initially asserted the Timeline event using the JPA repository, but thought I'd better use the REST API after recent transaction woes!